### DISCUSSION TITLE
Is indicator_create_v1 function support bulk create ?

### DISCUSSION THREAD

**Discussion by donggyu-woo** [CrowdStrike/falconpy#1173] [Q&A] [answered]
Hello 

The more IoCs you add, the more you increase the code runtime.
I don't see it in the docs, but does it support Bulk Create as well?

If there is another way, could you please let me know?

**Comment by jshcodes** [comment]
Hi @uyggnodoow -

The body payload for this operation [defines the indicators to be created as a list](https://falconpy.io/Service-Collections/IOC.html#uber-class-example-5). You can create multiple IOCs with a single API call by leveraging the Uber Class or by providing a list of dictionaries to the `indicators` keyword when using the Service Class. 

Thank you for the question! 😄

**Comment by donggyu-woo** [comment]
Hi @jshcodes 

Thanks for answer. I will test it.

**Comment by donggyu-woo** [comment]
@jshcodes 

I ran it like below, but it failed to work.
Any idea what could be the problem?

The code looks like this 

```python
credentials = {
    'client_id': config['crowdstrikeClientId'],
    'client_secret': config['crowdstrikeClientSecret']
}
cs = APIHarnessV2(creds=credentials)

   ...

body = {
    'comment': 'crowdstrikeEDR',
    'indicators': indicators
}
response = cs.command(
    "indicator_create_v1",
    ignore_warnings=True,
    retrodetects=False,
    body=body
)
```

The body value looks like this  (I deleted the hash value.)

```json
{'comment': 'crowdstrikeEDR', 'indicators': [{'action': 'detect', 'applied_globally': True, 'platforms': ['mac', 'windows', 'linux'], 'severity': 'High', 'source': 'v2', 'type': 'SHA256', 'value': ''}, {'action': 'detect', 'applied_globally': True, 'platforms': ['mac', 'windows', 'linux'], 'severity': 'High', 'source': 'v2', 'type': 'SHA256', 'value': ''}, {'action': 'detect', 'applied_globally': True, 'platforms': ['mac', 'windows', 'linux'], 'severity': 'High', 'source': 'v2', 'type': 'SHA256', 'value': ''}, {'action': 'detect', 'applied_globally': True, 'platforms': ['mac', 'windows', 'linux'], 'severity': 'High', 'source': 'v2', 'type': 'SHA256', 'value': ''}, {'action': 'detect', 'applied_globally': True, 'platforms': ['mac', 'windows', 'linux'], 'severity': 'High', 'source': 'v2', 'type': 'SHA256', 'value': ''}]}
```

This is the result of running 

```text
Status Code : 400
One or more indicators have a warning or invalid input
```

**Comment by donggyu-woo** [comment, marked as answer]
Hello @jshcodes 

I solved the problem. The cause was adding duplicate hash values.

Thank you.
